skaffold.yaml

```diff
@@ -31,6 +31,10 @@ deploy:
           controller:
             service:
               type: ClusterIP
+            ingressClass:
+              name: nginx
```

Comment:
shouldn't we use the ingress.class: haproxy instead as we switched to use haproxy as ingress controller.

Reply:
No, these are helm values for ingress-nginx